api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -89,4 +89,174 @@ enum Modulation {
+  PSK8 = 5;
+
+  // 16-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK16 = 6;
+
+  // 32-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK32 = 7;
+
+  // 64-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK64 = 8;
+
+  // 128-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK128 = 9;
+
+  // 256-Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying
+  PSK256 = 10;
+
+
+  // Differential Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying#Differential_phase-shift_keying_(DPSK)
+  DPSK = 11;
+
+  // Optical Differential Phase Shift Keying. https://www.bell-labs.com/our-research/publications/203155/
+  ODPSK = 12;
+
+  // Offset Quadrature Phase Shift Keying. https://en.wikipedia.org/wiki/Phase-shift_keying#Offset_QPSK_(OQPSK)
+  OQPSK = 13;
+
+  // Asynchronous Quadrature Phase Shift Keying.
+  AQPSK = 14;
+
+
+  // 8-Phase Amplitude Modulation. https://en.wikipedia.org/wiki/Quadrature_amplitude_modulation
+  QAM8 = 15;
+
+  // 16-Phase Amplitude Modulation. https://en.wikipedia.org/wiki/Quadrature_amplitude_modulation
+  QAM16 = 16;
+
+  // 32-Phase Amplitude Modulation. https://en.wikipedia.org/wiki/Quadrature_amplitude_modulation
+  QAM32 = 17;
+
+  // 64-Phase Amplitude Modulation. https://en.wikipedia.org/wiki/Quadrature_amplitude_modulation
+  QAM64 = 18;
+
+  // 128-Phase Amplitude Modulation. https://en.wikipedia.org/wiki/Quadrature_amplitude_modulation
+  QAM128 = 19;
+
+  // 256-Phase Amplitude Modulation. https://en.wikipedia.org/wiki/Quadrature_amplitude_modulation
+  QAM256 = 20;
+
+
+  // Minimum Shift Keying. https://en.wikipedia.org/wiki/Minimum-shift_keying
+  MSK = 21;
+
+  // Gaussian Minimum Shift Keying. https://en.wikipedia.org/wiki/Minimum-shift_keying#Gaussian_minimum-shift_keying
+  GMSK = 22;
+
+
+  // Amplitude modulation (generic)
+  AM = 23;
+
+  // Frequency modulation (generic)
+  FM = 24;
+
+  // Phase modulation (generic)
+  PM = 25;
+
+  // Pulse-code modulation (generic)
+  PCM = 26;
+}
+
+// Line coding applied during modulation. Line coding is the technique for modeling binary 0 and 1
+// within a modulated waveform.
+enum LineCoding {
+  // Non-return-to-zero level. https://en.wikipedia.org/wiki/Non-return-to-zero
+  NRZ_L = 0;
+
+  // Non-return-to-zero mark. https://en.wikipedia.org/wiki/Non-return-to-zero
+  NRZ_M = 1;
+
+  // Non-return-to-zero space. https://en.wikipedia.org/wiki/Non-return-to-zero
+  NRZ_S = 2;
+
+  // Return-to-zero. https://en.wikipedia.org/wiki/Return-to-zero
+  RZ = 3;
+
+  // Bipolar return-to-zero level. https://en.wikipedia.org/wiki/Bipolar_encoding
+  BP_L = 4;
+
+  // Bipolar return-to-zero mark. https://en.wikipedia.org/wiki/Bipolar_encoding
+  BP_M = 5;
+
+  // Bipolar return-to-zero space. https://en.wikipedia.org/wiki/Bipolar_encoding
+  BP_S = 6;
+}
+
+// A type of Viterbi coding applied to a bitstream.
```

Comment:
https://en.wikipedia.org/wiki/Convolutional_code

Reply:
"Viterbi convolutional coding"

Reply:
Oh! For this one and other comments like it, just "convolutional coding" is more accurate. Viterbi decoding is a special type of "convolutional decoding" and these parameters actually have nothing to do with the Viterbi algorithm, but the underlying convolutoinal code.